src/components/video.tsx

```diff
@@ -118,7 +120,7 @@ export function getVideoProps(allProps: VideoProps, state: { asset?: Asset }) {
       if (playbackId) {
         props.src = `https://stream.mux.com/${playbackId}.m3u8`;
 
-        if (!poster) {
+        if (poster == undefined) {
```

Comment:
I feel like normally I use `typeof poster === 'undefined'` to avoid problems with type coercion with `null`, but effectively that won't happen here, right?

Reply:
indeed it's true for null or undefined, we still want to set the Mux poster on null but not on an empty string for example